config/default.yml

```diff
@@ -212,6 +212,20 @@ Style/CollectionMethods:
     detect: 'find'
     find_all: 'select'
 
+# Use ` or %x around command literals.
+Style/CommandLiteral:
+  EnforcedStyle: mixed
```

Comment:
Right. I might revisit those rules in the guide, though. Rarely had to write multi-line command literals, so I didn't really dwell on this much. :-)